source/docs/beta/tasks/task-4-new-features.rst

```diff
@@ -0,0 +1,29 @@
+Task 4: Testing New Features
+============================
+
+Task: Test any new features or heavily modified code
+
+Overview
```

Comment:
Why don't we just link or replace this page with the New for 2020 document?

Reply:
Because the list of things we want to call attention to is very likely a subset of what's in that document.

Reply:
I think a link is good, but agree that it shouldn't replace the smaller priority list. 

Recommend adding the synchronous PID Controller and C++ units

Reply:
Added to list per Joe's suggestion and added link to page with full list.